sql/mob_pools.sql

```diff
@@ -3478,7 +3478,7 @@ INSERT INTO `mob_pools` VALUES (3422,'Ruphuabo','Ruphuabo',194,0x000083040000000
 INSERT INTO `mob_pools` VALUES (3423,'Ryy_Qihi_the_Idolrobber','Ryy_Qihi_the_Idolrobber',360,0x0000390400000000000000000000000000000000,6,6,2,240,100,0,1,0,1,2,0,32,0,159,0,0,0,0,0,360,360);
 INSERT INTO `mob_pools` VALUES (3424,'Sabertooth_Tiger','Sabertooth_Tiger',242,0x0000340100000000000000000000000000000000,1,1,7,240,100,0,1,0,0,0,0,0,238,131,0,0,0,0,0,242,242);
 INSERT INTO `mob_pools` VALUES (3425,'Sable-tongued_Gonberry','Sable-tongued_Gonberry',243,0x0000C60400000000000000000000000000000000,4,4,6,240,100,0,1,0,1,18,0,32,167,157,0,0,2,0,0,243,243);
-INSERT INTO `mob_pools` VALUES (3426,'Sabotender','Sabotender',212,0x0000740100000000000000000000000000000000,2,2,7,360,100,0,1,0,0,0,0,0,6461,131,0,0,0,0,0,212,212);
+INSERT INTO `mob_pools` VALUES (3426,'Sabotender','Sabotender',212,0x0000740100000000000000000000000000000000,2,2,7,360,100,0,1,0,1,0,0,0,6461,131,0,0,0,0,0,212,212);
```

Comment:
pr doesn't hinge on it but could I comment at the end of this row telling ppl that the linking is adjusted in script? Just so they don't undo your work here thinking its wrong.

Reply:
Done.